pysyncrosim/session.py

```
@@ -436,110 +450,168 @@ def install_conda(self, software="miniforge"):
+            e = ps.environment._environment()
+            if e.program_directory.item() is not None:
+                # Running inside SyncroSim environment, skip validation
+                return "mono"
+
+        mono_path = shutil.which("mono")
+        if mono_path is None:
+            raise RuntimeError(
+                "Mono is required to run SyncroSim on Linux but was not found.\n"
+                "Install Mono: sudo apt-get install mono-complete\n"
+                "Or specify custom path: Session(location=..., mono_path=...)"
+            )
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)